File: SDKs/nexmo.com/voice/1.3.8/python/src/sdk/models/shared/createcallresponse.py
from dataclasses import dataclass, field
from typing import Enum,Optional
from dataclasses_json import dataclass_json
from . import direction_enum


@dataclass_json
@dataclass
class CreateCallResponse:
    conversation_uuid: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'conversation_uuid' }})
    direction: Optional[direction_enum.DirectionEnum] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'direction' }})
    status: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'status' }})
    uuid: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'uuid' }})
    

File: SDKs/nexmo.com/voice/1.3.8/typescript/src/sdk/models/shared/createcallresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { DirectionEnum } from "./directionenum";



export class CreateCallResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=conversation_uuid" })
  conversationUuid?: string;

  @SpeakeasyMetadata({ data: "json, name=direction" })
  direction?: DirectionEnum;

  @SpeakeasyMetadata({ data: "json, name=status" })
  status?: string;

  @SpeakeasyMetadata({ data: "json, name=uuid" })
  uuid?: string;
}
